meta1v2/meta1_backend_services.c

```diff
@@ -985,19 +996,19 @@ meta1_backend_services_link (struct meta1_backend_s *m1,
 }
 
 GError*
-meta1_backend_services_poll(struct meta1_backend_s *m1,
+meta1_backend_services_renew(struct meta1_backend_s *m1,
 		struct oio_url_s *url, const char *srvtype,
-		gboolean dryrun, gboolean autocreate,
+		const char *last, gboolean autocreate,
 		gchar ***result)
 {
-	if (!result) return SYSERR("BUG: invalid output array");
+	if (!srvtype)
+		return SYSERR("Missing service type");
+	if (!result)
+		return SYSERR("Missing output variable");
```

Comment:
If `srvtype` is NULL or empty, it's a client error. But if `result` is NULL, I think it's a coding error, and thus deserves an `assert`.